packages/core/src/renderTexture/RenderTextureSystem.ts

```diff
@@ -34,7 +35,13 @@ const tempRect2 = new Rectangle();
  */
 export class RenderTextureSystem implements ISystem
 {
-/* eslint-enable max-len */
+    /** @ignore */
+    static extension: ExtensionMetadata = {
+        type: ExtensionType.RendererSystem,
+        name: 'renderTexture',
+    };
+
+    /* eslint-enable max-len */
```

Comment:
```suggestion
```

Reply:
This is used to ignore the long table above

Reply:
:facepalm:

yep i misread that one